packages/website/src/components/Schedule/Schedule.tsx

```diff
@@ -4,53 +4,61 @@ import { Button } from '@coderscamp/ui/components/Button';
 import { Center } from '@coderscamp/ui/components/Center';
 import { SimpleGrid } from '@coderscamp/ui/components/SimpleGrid';
 import { Typography } from '@coderscamp/ui/components/Typography';
+import { useDisclosure } from '@coderscamp/ui/hooks/useDisclosure';
```

Comment:
🤔 To share our modals across sections we can add them to `_app.tsx` and then use context to display them from particular sections